hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -69,15 +66,20 @@
   public static final String SUFFIX_READ_OPTIMIZED_TABLE = "_ro";
 
   protected final HiveSyncConfig cfg;
-  protected HoodieHiveClient hoodieHiveClient = null;
+  protected AbstractHiveSyncHoodieClient hiveClient = null;
   protected String snapshotTableName = null;
   protected Option<String> roTableName = null;
 
   public HiveSyncTool(HiveSyncConfig cfg, HiveConf configuration, FileSystem fs) {
     super(configuration.getAllProperties(), fs);
 
     try {
-      this.hoodieHiveClient = new HoodieHiveClient(cfg, configuration, fs);
+      if (cfg.isAwsGlueMetaSyncEnabled) {
+        LOG.info("Syncing Hudi Table with the AWS Glue Metastore");
```

Comment:
don't think we should log this at constructor stage. it's confusing as it hasn't done actual sync yet.

Reply:
you can log it in the concrete class sync method instead